plugins/versionpress/src/Utils/QueryLanguageUtils.php

```diff
@@ -225,16 +225,16 @@ public static function createGitLogQueryFromRule($rawRule)
             }
 
             if (substr($key, 0, 5) === 'x-vp-') {
-                $prefix = '';
+                $key = str_replace('x-vp-', 'X-VP-', $key);
             } else {
                 if (substr($key, 0, 3) === 'vp-') {
-                    $prefix = '\(X-\)\?';
+                    $key = '\(X-\)\?' . str_replace('vp-', 'VP-', $key);
                 } else {
-                    $prefix = '\(X-VP-\|VP-\)';
+                    $key = '\(X-VP-\|VP-\)' . $key;
```

Comment:
I stayed with capitalized letters here and changed `queryLanguageUtilsGeneratesCorrectGitLogQuery` test accordingly to follow `VP-Action` example (also capitalized letters) and made some small changes to allowing test to pass (`assertEquals()` is now case sensitive). But it can be also switched to lowercase characters.